python/hsfs/engine/spark.py

```diff
@@ -214,12 +214,14 @@ def convert_to_default_dataframe(self, dataframe):
             lowercase_dataframe = dataframe.select(
                 [col(x).alias(x.lower()) for x in dataframe.columns]
             )
-            nullable_schema = copy.deepcopy(lowercase_dataframe.schema)
-            for struct_field in nullable_schema:
-                struct_field.nullable = True
-            lowercase_dataframe = self._spark_session.createDataFrame(
-                lowercase_dataframe.rdd, nullable_schema
-            )
+            # for streaming dataframes this will be handled in DeltaStreamerTransformer.java class
+            if not lowercase_dataframe.isStreaming:
```

Comment:
We might need the same change in the java engine here https://github.com/logicalclocks/feature-store-api/blob/master/java/src/main/java/com/logicalclocks/hsfs/engine/SparkEngine.java#L795